<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayer</class>
 <widget class="QMainWindow" name="MediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RailwayMediaPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="Camera1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(148, 171, 255);

</string>
    </property>
    <property name="text">
     <string>Камера 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 218, 166);
</string>
    </property>
    <property name="text">
     <string>Аннотация 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Speed1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 171, 255);
</string>
    </property>
    <property name="text">
     <string>х0,2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Play">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 166);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="Slider">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 171, 255);
border-radius: 10px</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Camera2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(148, 171, 255);
</string>
    </property>
    <property name="text">
     <string>Камера 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 218, 166);
</string>
    </property>
    <property name="text">
     <string>Аннотация 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Camera3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(148, 171, 255);
</string>
    </property>
    <property name="text">
     <string>Камера 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 218, 166);
</string>
    </property>
    <property name="text">
     <string>Аннотация 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 218, 166);
</string>
    </property>
    <property name="text">
     <string>Аннотация 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Camera4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(148, 171, 255);

</string>
    </property>
    <property name="text">
     <string>Камера 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Speed2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 171, 255);
</string>
    </property>
    <property name="text">
     <string>х1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Speed3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 171, 255);
</string>
    </property>
    <property name="text">
     <string>х10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>520</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 166);
</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QVideoWidget" name="Video1" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 20px</string>
    </property>
   </widget>
   <widget class="QVideoWidget" name="Video2" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 20px</string>
    </property>
   </widget>
   <widget class="QVideoWidget" name="Video4" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 20px</string>
    </property>
   </widget>
   <widget class="QVideoWidget" name="Video3" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 20px</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 171, 255);
border-radius: 10px</string>
    </property>
    <property name="text">
     <string>TelegramID</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="cursorPosition">
     <number>10</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>Label</zorder>
   <zorder>Camera1</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>Video1</zorder>
   <zorder>Video2</zorder>
   <zorder>Video4</zorder>
   <zorder>Video3</zorder>
   <zorder>Camera3</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>Play</zorder>
   <zorder>Speed1</zorder>
   <zorder>Speed2</zorder>
   <zorder>Speed3</zorder>
   <zorder>Slider</zorder>
   <zorder>Send</zorder>
   <zorder>Camera4</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>Camera2</zorder>
   <zorder>LineEdit</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/res.qrc"/>
 </resources>
 <connections/>
</ui>
